crates/hulk_nao/src/camera.rs

```diff
@@ -3,25 +3,30 @@ use std::{
     sync::Arc,
     time::Duration,
 };
-use tokio::sync::watch::{self, Receiver, Sender};
 
 use color_eyre::{
     eyre::{bail, Context},
     Result,
 };
 use nao_camera::{reset_camera_device, Camera as NaoCamera, Parameters, PollingError};
-use parking_lot::{Mutex, RwLock};
+use parking_lot::Mutex;
 use types::{camera_position::CameraPosition, ycbcr422_image::YCbCr422Image};
+use watch::WatchSender as Sender;
 
 pub struct Camera {
-    camera: RwLock<Option<NaoCamera>>,
-    path: PathBuf,
+    camera: Mutex<CameraHardware>,
+    image_sender: Sender<Option<YCbCr422Image>>,
```

Comment:
Why is the item type `Option`? Did I overlook the case where you send a `None`? Can we remove the `Option`?

Reply:
The channel needs to be initialized with something that is defaultable.